internal/dataplane/kong_client.go

```diff
@@ -546,6 +552,7 @@ func (c *KongClient) tryRecoveringWithFallbackConfiguration(
 	// TODO: https://github.com/Kong/kubernetes-ingress-controller/issues/6082
 	// Expose Prometheus metrics for fallback configuration parsing result.
 
+	c.cacheBrokenObjectList(brokenObjects)
```

Comment:
While it still might be useful to expose "broken" objects (these are the ones reported by the Gateway as broken in flattened errors), more important from the `FallbackConfiguration` perspective would be to expose what objects were excluded (https://github.com/Kong/kubernetes-ingress-controller/blob/main/internal/dataplane/fallback/fallback.go#L51) or backfilled (https://github.com/Kong/kubernetes-ingress-controller/blob/main/internal/dataplane/fallback/fallback.go#L93).

I think we can change `GenerateBackfillingBrokenObjects` and `GenerateExcludingBrokenObjects` methods to return affected objects so we can use them for reporting them to the diagnostics server.

Reply:
Fair enough. Make this a partial solution and handle those changes in a separate PR?